packages/actions/test/unit/helpers.utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from "chai"
+import { fakeCircuitsData } from "../data/samples"
+import { extractPoTFromFilename, extractPrefix, formatZkeyIndex } from "../../src"
+
+describe.skip("Utils", () => {
```

Comment:
Please, be sure to motivate the skip using comments and put a todo to avoid missing these tests in the future.

Reply:
My bad here committing this with skip, I only added skip to not run the tests while working on setup